### ISSUE TITLE
[1.20.1 Bug]: Can't use drawers from Functional Stoarge as a dragon

### ISSUE THREAD

**Issue by zevric** [DragonSurvivalTeam/DragonSurvival#462]
### ■ Your Discord ID or other contacts

Zevric

### ■ Dragon Survival version

DragonSurvival-1.20.1-09.01.2024

### ■ Forge/Neoforge version

Forge 47.2.17

### ■ Single game or server?

Single-player (Default)

### ■ Describe the bug

When i left click a storage drawer, even with no items equipped on claws, i can't out items into a storage drawers. It makes the drawer recognize and item was used on it and the display reflects that, but i can't put anything in manually or even take anything out manually.

### ■ Crash Report File and Logs

N/A

**Comment by BlackAures1** [comment]
https://github.com/DragonSurvivalTeam/DragonSurvival/issues/388 not same?

**Comment by zevric** [comment]
No, it's not the same issue. The issue i have persists even without tools in the claw slows as stated in the description. I have to hold crouch, which oddly makes the camera move upwards, and spam click the drawer to get anything out. There's no double item inserting or removing.
